### DISCUSSION TITLE
Problem with large amount of attachmets

### DISCUSSION THREAD

**Discussion by jumat-dt** [PrestaShop/PrestaShop#34492] [Ideas]
### The problem

I have a problem with editing products in the backoffice, due to a large number of attachments (13631).
It takes 20 seconds to load the product before editing.
8.0.4 version.

### The solution

When I removed all attachments, it takes 1 second.

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
Which product page you're using? Is the one available behind the feature flag or the old one? If the old one, could you check how the page works if you enable the experimental one?
